Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -133,11 +138,17 @@ class NavigationHeader extends React.Component<DefaultProps, Props, any> {
     });
 
     return (
-      <View style={[ styles.appbar, style ]} {...viewProps}>
+      <Animated.View style={[
+          styles.appbar,
+          { height: APPBAR_HEIGHT + this.props.statusBarHeight },
```

Comment:
@corbt Yeah. It's would be possible to support animating the size as well as a separate `appbarHeight` prop, which many people seem to be requesting. Right now it's not simple to customize `statusbarHeight` and `appbarHeight` separately since they are a single element. I think the performance implications of one extra view is negligible here.